activities/PHQ-9/phq9_schema.jsonld

```diff
@@ -2,37 +2,31 @@
     "@context": [ "https://raw.githubusercontent.com/sanuann/schema-standardization/master/contexts/generic.jsonld",
         "https://raw.githubusercontent.com/sanuann/schema-standardization/master/activities/PHQ-9/phq9_context.jsonld"
     ],
-    "@type": "https://schema.metadatacenter.org/core/Template",
-    "title": "PHQ-9 form schema",
-    "schema:description": "PHQ-9 assessment schema",
+    "@type": "https://raw.githubusercontent.com/sanuann/schema-standardization/master/schemas/Activity.jsonld",
+    "@id": "#phq9_schema",
+    "skos:prefLabel": {
+        "en": "PHQ-9 Assessment"
```

Comment:
can't we use default language as en for this schema? and reduce this to just a string?

Reply:
yes. I have used the default language thing. but thought it needs to be written this way. just corrected it.